src/transformers/tokenization_utils.py

```diff
@@ -176,70 +213,104 @@ def values(self):
     def items(self):
         return self.data.items()
 
-    def char_to_token_offsets(self, sentence: int, char: int) -> Tuple[int, int]:
-        """
-        Find the Offsets of the token containing the character at the specified position
+    def token_to_word(self, key: Union[int, slice], batch: int = 0) -> Union[int, List[int]]:
```

Comment:
I would rather rename the parameter `key` as BatchEncoding is Dictionary, it seems misleading as `key` here related to indices. 

Also, using a default argument value as indice for array indexation sounds very strange to me, as it reverse the actual order you would index the underlying structure [batch][key] -> `token_to_word(key, batch)` (looks like Fortran indexing schema x) )

Reply:
Indeed.

Did that because if the user is just indexing in a single batch (from `encode` for instance), it's a bit strange to ask him to give the batch index.

I'll fix it by doing like python `range` method (optional first argument).